client.go

```diff
@@ -68,7 +68,23 @@ func DialAddrEarly(
 	tlsConf *tls.Config,
 	config *Config,
 ) (EarlySession, error) {
-	sess, err := dialAddrContext(context.Background(), addr, tlsConf, config, true)
+	sess, err := DialAddrEarlyContext(context.Background(), addr, tlsConf, config)
+	if err != nil {
+		return nil, err
+	}
+	utils.Logger.WithPrefix(utils.DefaultLogger, "client").Debugf("Returning early session")
+	return sess, nil
```

Comment:
You already have that log statement in `DialAddrEarlyContext`. `DialAddrEarly` can be as simple as just a `return DialAddrEarlyContext(context.Background(), ...)`.

Reply:
My bad. Just fixed it